maro/simulator/scenarios/data_center/physical_machine.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, id: int, cpu_cores_capacity: int, memory_capacity: int):
         self.cpu_cores_capacity: int = cpu_cores_capacity
         self.memory_capacity: int = memory_capacity
         # PM resource.
-        self._live_vms: Set(int) = set()
+        self._live_vms: Set[int] = set()
         self.cpu_allocation: int = 0
         self.memory_allocation: int = 0
         self._cpu_utilization: float = 0.0
```

Comment:
Add cpu_utilization and energy_consumption to snapshot_list, then we don't need these two series?